test/selenium/lib/view/signed_out/view.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var webdriver = require('selenium-webdriver');
+var SignedOutPageAccessor = require('./accessors.js');
+
+function SignedOutPageView(driver) {
+    this.driver = driver;
+    this.accessors = new SignedOutPageAccessor(this.driver);
+};
+
+SignedOutPageView.prototype = {
+    confirmPage: function() {
```

Comment:
I'm not too sure how to describe this function as a user's action. What did you have in mind?